packages/manager/apps/pci-databases-analytics/src/__tests__/helpers/wrappers/RouterWithQueryClientWrapper.tsx

```diff
@@ -77,6 +77,7 @@ export const RouterWithQueryClientWrapper = ({
         getEnvironment: vi.fn(),
         setUniverse: vi.fn(),
         setApplication: vi.fn(),
+        setUser: vi.fn(),
```

Comment:
This is not a blocking comment.
But I doubt that all these function / data are used in the tests suites of this project and could be cleaned